app/assets/stylesheets/components/notices.scss

```diff
@@ -19,6 +19,6 @@
   }
 
   &--info {
-    background: var(--accent-brand-a10);
```

Comment:
This is much more flexible as we're no longer constrained to only one opacity-related variable that's been available.